cnv/cnv-attaching-vm-multiple-networks.adoc

```diff
@@ -0,0 +1,28 @@
+[id="attaching-to-multiple-networks"]
+= Attaching a virtual machine to multiple networks
+include::modules/cnv_document_attributes.adoc[]
+:context: cnv-attaching-multiple-networks
+toc::[]
+
+{ProductName} provides Layer-2 networking capabilities that allow you to connect
+virtual machines to multiple networks. You can import virtual machines with
+existing workloads that depend on access to multiple interfaces. You can also
+configure a PXE network so that you can boot machines over the network.
+
+To get started, a network administrator configures a NetworkAttachmentDefinition
+of type `cnv-bridge`. Then, users can attach Pods, VMIs, and VMs to the
+bridge network. From the {ProductName} web console, you can create a vNIC
+that refers to the bridge network.
+
+include::modules/cnv-networking-glossary.adoc[leveloffset=+1]
```

Comment:
No such file exists.

Reply:
I cannot stand github, but you can find the file in the "File changed" tab. The link to it https://github.com/openshift/openshift-docs/pull/14351/files#diff-df6e4689984821cbe8be31fb4871b266 would expire if Pan squashes the commit in the future.